front/app/containers/Admin/projects/project/analysis/Tags/index.tsx

```diff
@@ -81,6 +82,19 @@ const Tags = () => {
     filters: omit(filters, 'tag_ids'),
   });
 
+  useEffect(() => {
```

Comment:
A bit of a complicated mechanism to scroll to the new tag but it's needed because we need to refetch the list of tags first and this is the way to know when this has happened.